### ISSUE TITLE
`--show-path` behavior abnormal

### ISSUE THREAD

**Issue by boxie123** [astral-sh/rye#751]
### Steps to Reproduce

`rye config --show-path`

### Expected Result

Display the path of my configuration file

### Actual Result

```ps
> rye config --show-path
error: one or more required arguments were not provided
```

### Version Info

```
rye 0.25.0
commit: 0.25.0 (d8e00cea1 2024-02-19)
platform: windows (x86)
self-python: cpython@3.12
symlink support: true
uv enabled: true
```

**Comment by mitsuhiko** [comment]
This should be fixed on 0.26.0

**Comment by mitsuhiko** [comment]
Closing as fixed.
